YvesIraguha/lost-and-found-api
# #173977481 Feature enable delete lost or found items

### What does this PR do?

- Allow users to delete found or lost items from database

### Description of Task to be completed?

- Setting a route to handle delete items request.

- Send a `Delete` request to remove document from database.

### How should this be manually tested?

- Clone a repo `git clone`

- Then switch a branch by ` git checkout ft-delete-items-#173977481` .

- Run `npm install` to install all dependancies. And then run `npm run dev` to start the server.

- In postman send a delete request on `http://localhost:3000/api/v1/items/delete/_id` with a valid `_id` you should see a deleted document with message including the document number. Otherwise you should see an error indicating the nonexistence of document. 

### Any background context you want to provide?

- To be able to send a delete request you must login and provide token in header key as `auth-token` and `token` as value.

### What are the relevant pivotal tracker stories?
#173977481
